packages/core/src/commands/insertContentAt.ts

```diff
@@ -127,11 +127,11 @@ export const insertContentAt: RawCommands['insertContentAt'] = (position, value,
     }
 
     if (options.applyInputRules) {
-      tr.setMeta('applyInputRules', { from, text: newContent })
+      tr.setMeta('applyInputRules', { from, text: newContent.toString() })
```

Comment:
Hm, I see why you are doing this. But I think it is probably better to do it in the `else` branches on line :119 and :114 instead since those are the branches that does not attempt to convert it to a string (really 119 is the one that matters 114 should be a string)

Reply:
Also tests would be greatly appreciated

Reply:
that what I originally had, but I opted for this since I think we want `newContent` to retain it's original assignment for the `tr.replaceWith(from, to, newContent)` in line :121.

Reply:
since this is my first contribution to this repo, I don't know where everything is. can you point me to where tests for a `command` or `pasteRule` would go, since I don't see any preexisting ones, or perhaps I am missing them. @nperez0111 thank you!

Reply:
The tests would be in this file: https://github.com/ueberdosis/tiptap/blob/21aa96dee8deab1f439b7f655b8ed266a516a4cd/demos/src/Commands/InsertContent/React/index.spec.js#L1-L44

You can follow this guide to get your environment set up: https://github.com/ueberdosis/tiptap/blob/e597809c110011dd166e0cbb2488aee5a4434197/docs/overview/contributing.md#L22-L29


You are right about maintaining the call, I missed that.

I appreciate your contribution very much. If you need any help, I'm happy to 😄 !